herd/tests/instructions/AArch64.mixed/A157.litmus

```diff
@@ -0,0 +1,13 @@
+AArch64 A157
+(* Tests load acquire register, symbolic location, no offset*)
+(* Equivalent to A22 but testing mixed mode *)
+
+Variant=mixed
+
+{ 0:X1=x; uint64_t x; }
```

Comment:
I have run tests under litmus and have some suggestions. Generally, it is a good idea to check tests by using litmus, as litmus has different requirements, sometime stronger.
For this test:
```suggestion
{ uint64_t 0:X0; 0:X1=x; uint64_t x; }
```
Rationale: type declaration of a register commands its printing by litmus infrastructure. Here register X0 reads an unsigned integer of 64 bits and is more consistently printed as such, although it is not very noticeable here as the value is zero.